### PR TITLE
Updated path following cfp-speakers services refactoring.

### PR DESCRIPTION
**Pull request by jollivetc**
Updated path. App and test run.

Not sure about components/cfp-speaker/app/config.js addition.
Should ask to ndeloof who made the addition in the cfp-speaker app on commit https://github.com/devoxx/cfp-speaker/commit/7a866293e830d997c6557cc0e61ee66c6167fb3d

Maybe we should not rely on the one in the cfp-speakers dependency but on out own file.

### CODE REVIEW

**Comment by clakech** [comment]
je regarde

File: karma.conf.js [hunk 1]
```diff
@@ -13,8 +13,10 @@ files = [
     'app/components/angular-bootstrap/ui-bootstrap-tpls.js',
     'app/components/mousetrap/mousetrap.js',
     'app/components/ngInfiniteScroll/ng-infinite-scroll.js',
-    'app/components/cfp-speaker/app/scripts/services.js',
-    'app/components/cfp-speaker/app/scripts/eventbus.js',
+    'app/components/cfp-speaker/app/config.js',
```
Comment (clakech) [inline karma.conf.js]: config.js should be moved in cfp-speaker/app/

then change app/scripts/config.js -> endPoint variable value to baseUri ?

**Comment by jollivetc** [comment]
cfp-review service is using endPoint defined in app/scripts/config.js.
UserService from cfp-speakers in component is using baseUri in app/config.js

I think that we need to clean up and  use the same.

**Comment by clakech** [comment]
Yes. That is what I wanted to say in comment https://github.com/devoxx/cfp-review/pull/13#discussion_r5174623

**Comment by clakech** [comment]
change app/scripts/config.js -> endPoint variable value to baseUri

**Pull request merged**